### ISSUE TITLE
You can not browse for objects within various apps on Windows 10

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#6238]
When you open some windows applications on 10.0.14393 and try to navigate through objects with NVDA + shift + right or left, many objects are not read, or can not get the objects.
examples:
news app
Soon to open a news, you can not read the news with the screen operation or object exploration, and much less with tabe and arrows, the only way to read the news is by swiping the mouse pad to locatio news .
app money
Inside Money app when choosing a news, as news app can not read the text with the scroll across the screen, objects navigation, or arrows or tabe.
Other apps NVDA tains very difficult to read the objects to navigate tabe (navigation is unreliable, sometimes the information is read other not), and if you try to use the navigation objects NVDA gets stuck on only one object and does not read the others.
An application example of this form is downloaded health project (projeto saúde) of the Brazilian shop to try to register a new user. Most point out that this is a problem in many applications.

**Comment by josephsl** [comment]
Hi, Which version of NVDA are you using? Sounds like a popup has appeared, and if so, you need to select “OK”, “close” or what not to dismiss them. Thanks.

**Comment by fernando-jose-silva** [comment]
I am using the master 13481, there is not a popup

Thank you very much.

 

De: Joseph Lee [mailto:notifications@github.com] 
Enviada em: quinta-feira, 4 de agosto de 2016 18:53
Para: nvaccess/nvda <nvda@noreply.github.com>
Cc: fernando-jose-silva <fernandoj.silva21@gmail.com>; Author <author@noreply.github.com>
Assunto: Re: [nvaccess/nvda] You can not browse for objects within various apps on Windows 10 (#6238)

 

Hi, Which version of NVDA are you using? Sounds like a popup has appeared, and if so, you need to select “OK”, “close” or what not to dismiss them. Thanks.

**Comment by josephsl** [comment]
Hi, what happens if you restart NVDA and/or the apps in question? Thanks.

**Comment by fernando-jose-silva** [comment]
the result is the same.

Thank you very much.

 

De: Joseph Lee [mailto:notifications@github.com] 
Enviada em: quinta-feira, 4 de agosto de 2016 20:22
Para: nvaccess/nvda <nvda@noreply.github.com>
Cc: fernando-jose-silva <fernandoj.silva21@gmail.com>; Author <author@noreply.github.com>
Assunto: Re: [nvaccess/nvda] You can not browse for objects within various apps on Windows 10 (#6238)

 

Hi, what happens if you restart NVDA and/or the apps in question? Thanks.

**Comment by josephsl** [comment]
Hi, odd indeed, as I cannot reproduce this. Can you attach a log of this (making sure to set your log level to input/output from general settings) so we can try reproducing this? Thanks.

**Comment by fernando-jose-silva** [comment]
Follow the log with my navigation through objects Apps News and project health.
Navigation through the objects appears to be inconsistent sometimes read others do not, it's as if the screen was divided into several parts, and that the navigation objects can read these other parties is necessary to walk with the mouse around the screen and then yes this part it is read when navigating objects, and the screen operation.
[apps.txt](https://github.com/nvaccess/nvda/files/404845/apps.txt)

Navigation with tabe also sometimes does not read, and returning with tabe and then shift + tabe.

**Comment by leonardder** [comment]
@fernando-jose-silva: Could you please provide a status update?

**Comment by fernando-jose-silva** [comment]
I was not able to get the news with browsing for objects within the news app, however I was able to read the news after I opened it and tabbed browsing through the open window, this was previously not possible.

**Comment by Adriani90** [comment]
This works now as expected in NVDA's last alpha and the news app as well as many other apps. Object navigation works properly and I can also navigate news by arrow keys or table navigation with ctrl+alt+arrow keys. When pressing enter on a news article, I can read it in browse mode like any other website.
Closing as works for me.
